lib/components/app/print-layout.js

```diff
@@ -53,8 +54,24 @@ class PrintLayout extends Component {
     root.removeAttribute('class')
   }
 
+  /**
+   * Use one of the customIcons if provided,
+   * (similar to @opentriplanner/trip-form/ModeIcon)
+   * otherwise, fall back on TriMetLegIcon.
+   * TODO: Combine all custom icon rendering in one place.
+   */
+  customLegIcon = icons => {
```

Comment:
I was initially going to say that we should go ahead and immediately start using the TriMetLegIcon, but ideally we'd make this TriMetLegIcon be available via a slot in the overall application because we aren't always going to want to use the TriMetLegIcon in all of our deployments.